### PR TITLE
Modified OBJECTIVE_CYCLE for Multi-Tlumps and plots

### PR DESCRIPTION
**Pull request by davideziviani**

### CODE REVIEW

**Comment by ibell** [comment]
Can you please fix the conflict and I will merge this PR?

**Comment by davideziviani** [comment]
I should have fixed the conflict. Also I found a small typo.

**Comment by davideziviani** [comment]
@ibell  could you check if now you can merge it? I have another PR to make related to OBJECTIVE_CYCLE as I have integrated an iteration scheme over the rotational speed imposed by a motor (either with map or detailed model)

File: PDSim/core/core.py [hunk 1]
```diff
@@ -1114,43 +1128,68 @@ def OBJECTIVE_CYCLE(self, Td_Tlumps0, X, epsilon = 0.003, cycle_integrator = 'RK
             ###  -----------------------------------
             ###         The lump temperatures
             ###  -----------------------------------
-                
+    
             self.solvers.lump_eb_history.append([self.Tlumps, self.lumps_resid])
-            
-            if len(self.solvers.lump_eb_history) == 1:
+
+            if len(self.Tlumps) > 1:
```
Comment (davideziviani) [inline PDSim/core/core.py]: I need a more robust way of handling multi-lump. Probably a Broyden-type of solver.

**Comment by ibell** [comment]
Have you found a solution for this yet?  I would imagine for the recip that
this is a serious problem?

**Comment by davideziviani** [comment]
I am going to push the changes for the solver I wrote to handle both the multi-lump as well as the iteration of the rotational speed of the motor.

**Comment by davideziviani** [comment]
I am  going to send a new PR for this.

**Pull request closed without merging**